.E......E.....E..E.....F..E...........
Time: 0.086
There were 5 errors:
1) test32(RandoopTest0)java.io.FileNotFoundException: hi! (Is a directory)
	at java.io.FileOutputStream.open(Native Method)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:221)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:110)
	at java.io.PrintStream.<init>(PrintStream.java:208)
	at RandoopTest0.test32(RandoopTest0.java:623)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at RandoopTest.main(RandoopTest.java:8)
2) test26(RandoopTest0)java.io.FileNotFoundException: hi! (Is a directory)
	at java.io.FileOutputStream.open(Native Method)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:221)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:110)
	at java.io.PrintStream.<init>(PrintStream.java:208)
	at RandoopTest0.test26(RandoopTest0.java:464)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at RandoopTest.main(RandoopTest.java:8)
3) test21(RandoopTest0)java.io.FileNotFoundException: hi! (Is a directory)
	at java.io.FileOutputStream.open(Native Method)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:221)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:110)
	at java.io.PrintStream.<init>(PrintStream.java:208)
	at RandoopTest0.test21(RandoopTest0.java:341)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at RandoopTest.main(RandoopTest.java:8)
4) test19(RandoopTest0)java.io.FileNotFoundException: hi! (Is a directory)
	at java.io.FileOutputStream.open(Native Method)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:221)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:110)
	at java.io.PrintStream.<init>(PrintStream.java:208)
	at RandoopTest0.test19(RandoopTest0.java:300)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at RandoopTest.main(RandoopTest.java:8)
5) test12(RandoopTest0)java.io.FileNotFoundException: hi! (Is a directory)
	at java.io.FileOutputStream.open(Native Method)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:221)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:110)
	at java.io.PrintStream.<init>(PrintStream.java:208)
	at RandoopTest0.test12(RandoopTest0.java:170)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at RandoopTest.main(RandoopTest.java:8)
There was 1 failure:
1) test14(RandoopTest0)junit.framework.AssertionFailedError: Expected: <null> but was: class java.lang.Object
	at RandoopTest0.test14(RandoopTest0.java:233)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at RandoopTest.main(RandoopTest.java:8)

FAILURES!!!
Tests run: 32,  Failures: 1,  Errors: 5

Cycle length is4
before
after
Reached this
# of deadlocks detected 0
THE SIZE OF THE TRACE OMEN 32570
Exiting analysis
